### DISCUSSION TITLE
🫡 Deleting across multiple chats

### DISCUSSION THREAD

**Discussion by HithaishiDesai** [m4mallu/uniquify#7] [Ideas]
•The bot will index from multiple chats and then delete duplicates keeping unique files across channels

#extra suggestion
•if two files have exact size but different name bot deletes the one with [ @username or www.website.com] 

(These are same files renamed by other persons which are in channel)

Thanks 🙏🏻 ❤️

**Comment by m4mallu** [comment]
For..

1. The concept is good, but in practical little bit confusing and may took some time to complete the entire process ! Also have some confusions in some areas like, if bot has successfully indexed all the user dialogues and later if the reference(first indexed) chat  or files got a limitation? The situation is like a total loss. 

2. What if, two different files with the exact size ?

**Reply by HithaishiDesai** [reply]
1st one ✅

2nd issue was same as previous suggestion by other user (same file renamed with another name)


I thought as the files are same then both files size will be same maybe using that we can remove such files 


Later i read old explanation and I think it's not possible. 

Thanks sir 🙏🏻
